dnf/query.py

```diff
@@ -49,6 +49,15 @@ def downgrades(self):
         # :api
         return self.filter(downgrades=True)
 
+    def duplicated(self, exclude=[]):
```

Comment:
Actually `exclude` param is redundant now (we can use `.filter(name__not=excludes).duplicated()`)